### PR TITLE
Created the Achievements Page and added some Components for it

### PR DESCRIPTION
**Pull request by SergeyMOV**

### CODE REVIEW

File: components/ProgressBar.js [hunk 1]
```diff
@@ -16,27 +17,42 @@ export default function Progress({
...
     onReset(reset);
   }
-  function saveClick() {}
+  function handleSave(e) {
+    e.preventDefault();
+    localStorage.setItem("name", JSON.stringify(savedpercent));
+    localStorage.setItem("budget", JSON.stringify(savedbudget));
+  }
+
   const addFive = budget + addvalue + minusvalue + "€";
```
Comment (B4CKSL4SH-44) [inline components/ProgressBar.js:26]: You shouldn't turn your number into a string when you want to calculate with it...

File: components/ProgressBar.js [hunk 2]
```diff
@@ -76,6 +92,7 @@ const ButtonContainer = styled.div`
   width: 100%;
   justify-content: center;
   align-self: center;
+  padding-top: 5px;
```
Comment (B4CKSL4SH-44) [inline components/ProgressBar.js:95]: Please try to use relative units.

File: components/Valueinfofields.js [hunk 3]
```diff
@@ -24,12 +24,13 @@ export default function Valueinfo({budget, addvalue, minusvalue}) {
 
 const Valuefields = styled.section`
   display: flex;
+  margin-top: -5.2em;
   flex-direction: column;
   justify-content: flex-end;
   height: 0em;
```
Comment (B4CKSL4SH-44) [inline components/Valueinfofields.js:30]: Why height 0em?

**Review by B4CKSL4SH-44 — COMMENTED** [review]
Nice. Please take a look at my comments.

File: components/ProgressBar.js (hunk 1 above)
Comment (SergeyMOV) [reply components/ProgressBar.js:26]: I don't want to calculate it, but simply store it in localstorage.

File: components/ProgressBar.js (hunk 2 above)
Comment (SergeyMOV) [reply components/ProgressBar.js:95]: Ok, i will change it

File: components/Valueinfofields.js (hunk 3 above)
Comment (SergeyMOV) [reply components/Valueinfofields.js:30]: I will fix it

**Pull request merged**